skiko/src/iosMain/kotlin/org/jetbrains/skiko/context/MetalContextHandler.kt

```diff
@@ -2,10 +2,12 @@ package org.jetbrains.skiko.context
 
 import kotlinx.cinterop.useContents
 import org.jetbrains.skia.*
+import org.jetbrains.skiko.InternalSkikoApi
 import org.jetbrains.skiko.RenderException
 import org.jetbrains.skiko.SkiaLayer
 import org.jetbrains.skiko.redrawer.MetalRedrawer
 
+@OptIn(InternalSkikoApi::class)
 internal class MetalContextHandler(layer: SkiaLayer) : ContextHandler(layer, layer::draw) {
```

Comment:
Why do you need this annotation for `internal` class?

Reply:
This annotation helps to avoid warnings about usage of InternalSkikoApi functions inside this class.

Reply:
Right, that's just OptIn, not marking